architecture-building-systems/CityEnergyAnalyst
# I2749 update schemas.yml for get optimization decentralized folder building result heating/cooling activation

this PR resolves #2749 , also `doc_schemas.py` and `doc_glossary.py` are running again

to test:
- go to the branch
- run `cea workflow --workflow district-heating-system`
- check if columns in output file `get_optimization_decentralized_folder_building_result_heating_activation` are matched and explained in the glossary (https://city-energy-analyst.readthedocs.io/en/i2749-get_optimization_decentralized_folder_building_result_heating_activation/glossary.html)
- run `cea workflow --workflow district-heating-system`
- do the same for columns in `get_optimization_decentralized_folder_building_result_cooling_activation`
